.ci-operator.yaml

```diff
@@ -2,4 +2,4 @@ build_root_image:
   use_build_cache: true
   namespace: openshift
   name: release
-  tag: rhel-9-release-golang-1.21-openshift-4.16
+  tag: rhel-9-release-golang-1.22-openshift-4.17
```

Comment:
Should we jump directly up to: `release:rhel-8-release-golang-1.21-openshift-4.18 `

Reply:
Changed to `rhel-9-release-golang-1.22-openshift-4.18`